### ISSUE TITLE
vichan.net

### ISSUE THREAD

**Issue by RealAngeleno** [vichan-devel/vichan#676]
Who owns the vichan.net domain? It doesn't appear to be a normal cybersquatter, but it would be nice to have and to put vichan there as a software page.

**Comment by 3ss0w** [comment]
its owned by czaks. i think its the 1st domain for vichan

**Comment by RealAngeleno** [comment]
I'll see if he's still around. His other site linked on his profile @czaks was taken by a cybersquatter.

**Comment by 3ss0w** [comment]
He's inactive everywhere

**Comment by Mek101** [comment]
I suppose this is related to #668?

**Comment by Mek101** [comment]
Maybe the feature can be substituted by just making an API call to github?

**Comment by RealAngeleno** [comment]
> Maybe the feature can be substituted by just making an API call to github?

There's vichan.info, so the api call could easily go there. It's more of having an actual domain for vichan.

**Comment by chanadmins** [comment]
> I'll see if he's still around. His other site linked on his profile @czaks was taken by a cybersquatter.

There was a rumour he got arrested for bomb threats and he's been inactive on the internet for ages. Btw there's a new version of vichan imageboard on vichan.pl but it's not made by people who made original vichan (and a few later instances).

**Comment by 3ss0w** [comment]
> > I'll see if he's still around. His other site linked on his profile @czaks was taken by a cybersquatter.
> 
> There was a rumour he got arrested for bomb threats and he's been inactive on the internet for ages. Btw there's a new version of vichan imageboard on vichan.pl but it's not made by people who made original vichan (and a few later instances). pl.vichan.net was also revived a few years ago but went down again.

Yes, he was arrested for bomb threats but got let out. The new vichan version is made by some underage newfag that doesnt even know PHP (he admitted it in a post he removed). Czaks completely quit so theres no way you can contact him

**Comment by ctrlcctrlv** [comment]
As far as I know it's owned by @h00j, but he may have let it expire as he is not an “underage new[friend] that doesnt even know PHP”, so he may not be the current owner.
